tests/backup/backup_restore_basic_test.go

```diff
@@ -1442,3 +1442,188 @@ var _ = Describe("{BackupMultipleNsWithSameLabel}", func() {
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule backup [%s] with labels [%v]", scheduleName, namespaceLabel))
+			firstScheduleBackupName, err = GetFirstScheduleBackupName(ctx, scheduleName, orgID)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the name of the first schedule backup [%s]", firstScheduleBackupName))
+			err = NamespaceLabelBackupSuccessCheck(firstScheduleBackupName, ctx, bkpNamespaces, namespaceLabel)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying if the labeled namespace [%v] is backed up and checks for labels [%s] applied to backup [%s]", bkpNamespaces, namespaceLabel, firstScheduleBackupName))
+		})
+		Step("Restoring manual backup of application", func() {
+			log.InfoD("Restoring manual backup of application")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			restoreName = fmt.Sprintf("%s-%v", backupName, time.Now().Unix())
+			err = CreateRestore(restoreName, backupName, nil, destinationClusterName, orgID, ctx, nil)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying backup restore with name [%s] in default namespace", restoreName))
+			restoreNames = append(restoreNames, restoreName)
+		})
+		Step("Restoring first schedule backup of application", func() {
+			log.InfoD("Restoring first schedule backup of application")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			scheduleRestoreMapping = make(map[string]string)
+			backupScheduleNamespace, err := FetchNamespaceFromBackup(ctx, firstScheduleBackupName, orgID)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching namespaces %s from schedule backup %s", backupScheduleNamespace, firstScheduleBackupName))
+			restoredNameSpace := fmt.Sprintf("%s-%v", backupScheduleNamespace[0], time.Now().Unix())
+			scheduleRestoreMapping[backupScheduleNamespace[0]] = restoredNameSpace
+			customRestoreName := fmt.Sprintf("%s-%v", scheduleName, time.Now().Unix())
+			err = CreateRestore(customRestoreName, firstScheduleBackupName, scheduleRestoreMapping, destinationClusterName, orgID, ctx, nil)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of restoring scheduled backups %s in custom namespace %v", customRestoreName, scheduleRestoreMapping))
+			restoreNames = append(restoreNames, customRestoreName)
+		})
+	})
+	JustAfterEach(func() {
+		defer EndPxBackupTorpedoTest(contexts)
+		ctx, err := backup.GetAdminCtxFromSecret()
+		log.FailOnError(err, "Unable to px-central-admin ctx")
+		scheduleUid, err := GetScheduleUID(scheduleName, orgID, ctx)
+		log.FailOnError(err, "Error while getting schedule uid %s", scheduleName)
+		err = DeleteSchedule(scheduleName, scheduleUid, orgID)
+		dash.VerifySafely(err, nil, fmt.Sprintf("Verification of deleting backup schedule - %s", scheduleName))
+		time.Sleep(1 * time.Minute)
```

Comment:
Why is there `time.Sleep` here?

Reply:
Removed, added that for debugging purposes.